scripts/zones/Mamook/mobs/Hundredfaced_Hapool_Ja.lua

```diff
@@ -0,0 +1,64 @@
+-----------------------------------
+-- Area: Mamook
+--  Mob: Hundredfaced Hapool Ja
+-----------------------------------
+require("scripts/globals/spell_data")
+require("scripts/globals/titles")
+require("scripts/globals/status")
+mixins = { require("scripts/mixins/job_special") }
+-----------------------------------
+local ID = require("scripts/zones/Mamook/IDs")
+local entity = {}
+
+entity.onMobSpawn = function(mob)
+    local hundredfaced_hapool_ja = mob:getID()
+    mob:addListener("MAGIC_USE", "SPAWN_CLONES", function(mobArg, target, spell, action)
+        local spellId = spell:getID()
+        local hateTarget = GetMobByID(hundredfaced_hapool_ja):getTarget()
+
+        -- Utsusemi: Ichi (3 clones)
+        if spellId == xi.magic.spell.UTSUSEMI_ICHI then
+            for clone = hundredfaced_hapool_ja + 1, hundredfaced_hapool_ja + 3 do
+                if not GetMobByID(clone):isSpawned() then
+                    GetMobByID(clone):setSpawn(mob:getXPos()+math.random(1, 5), mob:getYPos(), mob:getZPos()+math.random(1, 5))
+                    SpawnMob(clone):updateEnmity(hateTarget)
+                end
+            end
+
+        -- Utsusemi: Ni/San (4 clones)
+        elseif spellId == xi.magic.spell.UTSUSEMI_NI or spellId == xi.magic.spell.UTSUSEMI_SAN then
+            for clone = hundredfaced_hapool_ja + 1, hundredfaced_hapool_ja + 4 do
+                if not GetMobByID(clone):isSpawned() then
+                    GetMobByID(clone):setSpawn(mob:getXPos()+math.random(1, 5), mob:getYPos(), mob:getZPos()+math.random(1, 5))
+                    SpawnMob(clone):updateEnmity(hateTarget)
+                end
+            end
+        end
+
+        for i = hundredfaced_hapool_ja + 1, hundredfaced_hapool_ja + 4 do
+            local pet = GetMobByID(i)
+            if pet:getCurrentAction() == xi.act.ROAMING then
+                pet:updateEnmity(target)
+            end
+        end
+    end)
+end
+
+entity.onMobEngaged = function(mob, target)
+    mob:castSpell(xi.magic.spell.UTSUSEMI_SAN, mob)
+end
+
+entity.onMobDeath = function(mob, player, isKiller)
+    local hundredfaced_hapool_ja = mob:getID()
+    for i = 1, 4 do DespawnMob(hundredfaced_hapool_ja + i) end
+end
+
+entity.onMobDespawn = function(mob)
+    local hundredfaced_hapool_ja = mob:getID()
+    for i = 1, 4 do DespawnMob(hundredfaced_hapool_ja + i) end
+    -- TODO: 72 hours taken from DB, need retail verification.
+    UpdateNMSpawnPoint(hundredfaced_hapool_ja)
+    mob:setRespawnTime(259200)
```

Comment:
Since there's nothing special about his spawn mechanics, you can just rely on the respawn time set in mob_groups and remove this and the changes to zone.lua and IDs.lua. Since it's a normal spawntype and has a respawn time set in the DB, all of this is handled automatically.